src/call-for-proposals.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Call for Proposals
+---
+
+<a href="https://pretalx.com/pycon-uk-2022/cfp" target="_blank">Submit your proposal</a>
+
+<p>We welcome proposals from <strong>everyone</strong>. You don't need to be an expert, or an expert speaker. Below you can find out what kind of proposals we're seeking, but if you're in any doubt whether you should submit a proposal, please see...</p>
+
+<a href="/call-proposals/why-you-should-submit-proposal/">Why you should submit a proposal</a>
+
+<p>The call for proposals is open until <strong>17:00 (Europe/London)</strong> on <strong>Saturday, 2nd July, 2022</strong>. Selections will be published by SELECTDAY SELECTMONTHWORD.</p>
+
+<a href="/call-proposals/proposal-submission-advice/">How to submit a more successful proposal</a>
+
+<p>Please note that there will be no sprints this conference. Sorry, we hope they'll be back next year!</p>
+
+<style>
+  .box_cfp figure {
+    float: right;
+    max-width: 250px;
+    margin-left: 1em;
+    margin-bottom: 1em;
+  }
+
+  .box_cfp h3 {
+    margin-top: 0;
+    margin-bottom: 0.5em;
+  }
+
+  .box_cfp p:nth-child(3) {
+    margin-top: 0.5em;
+  }
+
+  h2 {
+    margin-top: 3em;
+  }
+</style>
+
+<div class="box box_blue">
+  <figure>
+    <img src="/images/mikey_talk.jpg" alt="A person speaking into a microphone in front of a projected slide deck discussing 'Why are we here?'. Their laptop is visible and has a truly impressive collection of stickers on it.">
+  </figure>
+
+  <h3>Talks</h3>
+  <p><strong>Speak to the wonderful PyCon UK audience!</strong> Most talk slots are around 25 minutes, with some longer slots available in the schedule.</p>
```

Comment:
Can we remove "Most talk slots are around 25 minutes, with some longer slots available in the schedule."

Reply:
Perhaps replace with "Talk slots are 25 minutes"

Reply:
Updated.